hack/e2e.sh

```diff
@@ -2,11 +2,16 @@
 
 set -euo pipefail
 
+GO111MODULE = off
```

Comment:
Can't have spaces here. These should read: `export GO111MODULE=off`

I'd recommend running `shellcheck` on the scripts here. Also, if we have to change this anyway, maybe we should switch to using `git clone` to grab the repo, and avoid having to explicitly override these here.